src/acl_kernel_if.cpp

```diff
@@ -44,7 +44,9 @@ acl_process_autorun_profiler_scan_chain(unsigned int physical_device_id,
 // to verify that the hardware and HAL have an identical view of the CSR
 // address map.
 #define CSR_VERSION_ID_181 (3)
-#define CSR_VERSION_ID (4)
+#define CSR_VERSION_ID_2022_2 (4)
```

Comment:
On second thought this should be named after the first release in which it was introduced, which is [19.1](https://www.intel.com/content/www/us/en/software-kit/664561/intel-fpga-sdk-for-opencl-pro-edition-software-version-19-1.html).
```suggestion
#define CSR_VERSION_ID_19_1 (4)
```